build-system/test-configs/forbidden-terms.js

```diff
@@ -103,6 +103,79 @@ const forbiddenTermsGlobal = {
   'describes.*\\.only': {
     message: 'Please remove all instances of describes.only',
   },
+  // TODO(#35443) Reduce the number of files that use non default timeouts.
+  'this.timeout': {
+    message: 'Tests are not allowed to override the default timeout',
+    allowlist: [
+      // Markdown files.
+      'extensions/amp-a4a/rtc-documentation.md',
+      'extensions/amp-a4a/rtc-publisher-implementation-guide.md',
```

Comment:
These two files contain "this timeout", not "this.timeout". If you escape the period in the regex on line 107, this section can go away.

Reply:
Great suggestion